src/test/java/io/xpring/ilp/IlpIntegrationTests.java

```diff
@@ -0,0 +1,172 @@
+package io.xpring.ilp;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.interledger.spsp.server.grpc.CreateAccountResponse;
+import org.interledger.spsp.server.grpc.GetAccountResponse;
+import org.interledger.spsp.server.grpc.GetBalanceResponse;
+import org.interledger.spsp.server.grpc.SendPaymentResponse;
+
+import io.xpring.ilp.util.IlpConstants;
+import io.xpring.xrpl.XpringException;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+import java.util.Optional;
+
+public class IlpIntegrationTests {
+
+  /**
+   * Connector fields
+   */
+  // This should allow hermes to talk to the connector without exposing any ports
+  private static final Network network = Network.newNetwork();
+
+  private static final int CONNECTOR_PORT = 8080;
+
+  private static final int HERMES_PORT = 6565;
+
+  /**
+   *  Start up a connector from the nightly docker image
+   */
+  @ClassRule
+  public static GenericContainer connectorNode = new GenericContainer<>("interledger4j/java-ilpv4-connector:0.2.0") // FIXME use nightly
```

Comment:
Assign a TODO owner

Reply:
We should actually just use the `0.2.0` image, and make sure to update it whenever the SDK uses a new version of the connector